spec/dummy/config/database.yml

```diff
@@ -5,21 +5,26 @@
 #   gem 'sqlite3'
 #
 default: &default
-  adapter: sqlite3
+  adapter: postgresql
+  encoding: unicode
   pool: 5
-  timeout: 5000
+  host: localhost
```

Comment:
as above .. worth keeping things consistent with FO ?  .. just means can copy and paste from existing .env